containers/versions.yml

```diff
@@ -11,6 +11,7 @@ nginx_alpine_version:         "1.13.9"
 nginx_big_upload_version:     "1.4.0"
 openresty_alpine_fat_version: "1.13.6.1-1"
 postgres_alpine_version:      "10.3"
+postgres_version:             "9.6"
```

Comment:
Let's call this `debian_postgresql_version. It's the one that we use only on Debian and only because it's the latest one available.

Reply:
Done.